query/src/main/scala/filodb/query/exec/ScalarFixedDoubleExec.scala

```diff
@@ -0,0 +1,75 @@
+package filodb.query.exec
+
+import scala.concurrent.duration.FiniteDuration
+
+import monix.eval.Task
+import monix.execution.Scheduler
+import monix.reactive.Observable
+
+import filodb.core.DatasetRef
+import filodb.core.metadata.Column.ColumnType
+import filodb.core.query._
+import filodb.core.store.ChunkSource
+import filodb.query.{QueryConfig, QueryResponse, QueryResult}
+import filodb.query.Query.qLogger
+
+
+/**
+  * Exec Plans for fixed scalars which can execute locally without being dispatched
+  */
+case class ScalarFixedDoubleExec(id: String,
+                                 dataset: DatasetRef,
+                                 params: RangeParams,
+                                 value: Double,
+                                 limit: Int,
+                                 submitTime: Long = System.currentTimeMillis()) extends LeafExecPlan {
+
+
+  /**
+    * Sub classes should override this method to provide a concrete
+    * implementation of the operation represented by this exec plan
+    * node
+    */
+  override def doExecute(source: ChunkSource, queryConfig: QueryConfig)
+                        (implicit sched: Scheduler, timeout: FiniteDuration): ExecResult = ???
+
+  /**
+    * Args to use for the ExecPlan for printTree purposes only.
+    * DO NOT change to a val. Increases heap usage.
+    */
+  override protected def args: String = s"params = $params, value = $value"
+
+
+  override def execute(source: ChunkSource,
+                       queryConfig: QueryConfig)
+                      (implicit sched: Scheduler,
+                       timeout: FiniteDuration): Task[QueryResponse] = {
+    val columns: Seq[ColumnInfo] = Seq(ColumnInfo("timestamp", ColumnType.LongColumn),
+      ColumnInfo("value", ColumnType.DoubleColumn))
```

Comment:
Place this result schema in a re-usable static context instead of recreating for each invocation.